docker/build.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/sh
 
-echo "Cloning ${FAC_BRANCH}"
-git clone -b ${FAC_BRANCH} https://github.com/mroote/factorio-server-manager.git ${FAC_ROOT}
-echo "Creating build..."
-make gen_release
-echo "Copying build artifacts..."
-cp -v build/* /build/
\ No newline at end of file
+cd ..
+make build
+cp build/factorio-server-manager-linux.zip docker/factorio-server-manager-linux.zip
+
+cd docker
```

Comment:
```suggestion
(
  cd ..
  make build
  cp build/factorio-server-manager-linux.zip docker/factorio-server-manager-linux.zip
)
```

Using a subshell removes the requirement to cd back.